lib/screens/login/login.dart

```diff
@@ -39,40 +46,182 @@ class LoginScreen extends StatelessWidget {
+                              EmailField(emailController),
+                              SizedBox(height: 30 * heightFactor),
+                              PasswordField(passwordController),
+                              SizedBox(height: 10 * heightFactor),
+                            ],
+                          ),
+                        )),
+                    flex: 4,
+                  ),
+                  //Redirect to Forgot Password
+
+                  Expanded(
+                      flex: 1,
+                      child: Container(
+                        padding: EdgeInsets.only(right: D.horizontalPadding),
+                        alignment: Alignment.topRight,
+                        child: RichText(
+                          text: TextSpan(text: "Forgot Password?"),
+                        ),
+                      )),
+
+                  //LoginButton
+                  Expanded(
+                    child: Container(
+                      padding: EdgeInsets.only(right: D.horizontalPadding),
+                      alignment: Alignment.topRight,
+                      child: Container(
+                        decoration: BoxDecoration(
+                          borderRadius: BorderRadius.all(Radius.circular(30)),
+                          boxShadow: [
+                            BoxShadow(
+                              color: C.authButtonColor.withOpacity(0.2),
+                              blurRadius: 10,
+                              spreadRadius: 3,
+                              offset: Offset(0, 12),
+                            )
+                          ],
+                        ),
+                        child: RaisedButton(
+                          shape: RoundedRectangleBorder(
+                            borderRadius: BorderRadius.all(Radius.circular(30)),
+                          ),
+                          color: C.authButtonColor,
+                          onPressed: () => _login(context),
+                          child: Container(
+                            height: 60,
+                            width: 120,
+                            alignment: Alignment.center,
+                            child: Text(
+                              "Log In",
+                              style: TextStyle(
+                                color: C.primaryUnHighlightedColor,
+                                fontSize: 20 * heightFactor,
+                              ),
+                            ),
+                          ),
+                        ),
+                      ),
+                    ),
+                  ),
+                  //New here Text
+                  Expanded(
+                    flex: 2,
+                    child: Container(
+                      alignment: Alignment.centerRight,
+                      padding: EdgeInsets.only(right: (width / 13), top: 10),
+                      child: RichText(
```

Comment:
Don't use RichText here. The texts are supposed to be separated and the **Sign Up** text should have GestureDetector to redirect to SignupScreen. Refer the SignupScreen, how its done there.